src/bin/muxer/ivf.rs

```diff
@@ -45,6 +46,27 @@ impl Muxer for IvfMuxer {
 }
 
 impl IvfMuxer {
+  pub fn check_file(path: &str) -> Result<(), CliError> {
+    if fs::metadata(path).is_ok() {
+      print!("File '{}' already exists. Overwrite ? [y/N] ", path);
+      io::stdout().flush().unwrap();
+
+      let mut option_input = String::new();
+      io::stdin()
+        .read_line(&mut option_input)
+        .expect("Failed to read option.");
+
+      match option_input.as_str().trim() {
+        "y" | "Y" => return Ok(()),
+        _ => {
+          println!("Not overwriting, exiting.");
+          std::process::exit(0);
```

Comment:
You should return a CliError and forward it up.

Reply:
Sorry for the silly question, but instead of what ? ``std::process::exit(0);`` ?

Reply:
``` rust
return Err(CliError::NotOverwriting)
```